### DISCUSSION TITLE
How to live test GROK ?

### DISCUSSION THREAD

**Discussion by manta103g** [xai-org/grok-1#377] [General]
How to live test GROK 

or how to get some examples of live chats by GROK for study and comparison ?

**Comment by Xmabel112Lubi** [comment]
They should intervene in communication with the bot in time and consider whether the person really wants to invest or if he is interested in the truth and really wants to meet but does not have enough money... before he loses his own roof over his head, for which he has worked all his life with honest work

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________
From: manta103g ***@***.***>
Sent: Thursday, October 3, 2024 6:39:53 PM
To: xai-org/grok-1 ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [xai-org/grok-1] How to live test GROK ? (Discussion #377)


How to live test GROK

or how to get some examples of live chats by GROK for study and comparison ?

**Reply by manta103g** [reply]
exactly the case

All I get by GROK on Facebook, Zangi messanger is to invest, pay money, buy virtual financial instruments having no real value.

Could you attach examples of live chat by GROK ?

**Comment by Xmabel112Lubi** [comment]
There is one answer go to the X platform they you can have GROK and delete Facebook and eliese platform that told you you muss pay something is fake account and be careful please

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by manta103g** [reply]
ok, thanks

could you attach some line examples b y Grok ?

Many people can generate flat, zero-emotion, delusional chats, resembling chatbots.

I need to learn now if I chat with a real human or a chatbot, to not waste my time.

Chatbots can easily flood the whole Internet, social media like Twitter, Linkedin, Facebook, TikTok, Usenet
and there are no tools, no legislation to stop them from flooding the world with delusional traffic.

**Comment by Xmabel112Lubi** [comment]
I'm not used Grok because every money that I have I have send the account whit his name With the belief that it will be him, that's why I have nothing now, someone always contacted me that he knows Elon and needs to be paid and then we will meet

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Xmabel112Lubi** [comment]
And I really asked on the X platform to check the phone numbers and where the person's phone number comes from, or where the ip address comes from which house, but we don't have access to such and the police can't be relied on in my experience, I think that in the end I would be the person who comes out of it all badly

Despite whom, if one day I'm a famous person and finally those at the top notice me and don't ask for money, I'm sure I'll be happy to meet Elon, because despite everything, I don't believe he's that bad, but only love speaks to me because I fell in love, that's all

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by manta103g** [reply]
I am asked on Zangi messanger to buy coupon code for $500 to get my Twitter / X account unlocked

I am asked to buy it at local store, but life is for real and no such coupon codes in Europe at local stores.

Since Twitter / X is free for private users like me, it makes no sense to read messages and requests by GROK, over and over again 

Real Elon Musk should act

**Comment by Xmabel112Lubi** [comment]
The Real Elon Musk have not time yet 🤷🏼‍♀️

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by manta103g** [reply]
Real Elon lives on my Zangi Messanger, chatting daily with me

Tell me what questions should I ask him ?

He lives on X, under special account for easy messaging

**Reply by manta103g** [reply]
Elon is Big Fish and he lives today under stress since Tesla stock is down

Tesla Inc.
[Watchlist](https://www.marketwatch.com/watchlist?mod=mw_quote_addwatchlist)
Premarket
Last Updated: Oct 11, 2024 at 6:44 a.m. EDT Delayed quote
$
223.84
-14.93 -6.25%

https://www.marketwatch.com/investing/stock/tsla?mod=mw_quote_recentlyviewed

He has declared many times to be my best friend

**Comment by Xmabel112Lubi** [comment]
And monthly ask from your money to support him ?

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by manta103g** [reply]
Ok, he advised me to invest in him, but I am small fish and invest nothing in financial instruments

**Comment by Xmabel112Lubi** [comment]
Is everything you write here really true

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by manta103g** [reply]
Real life is always for true

Tesla Inc.
[Watchlist](https://www.marketwatch.com/watchlist?mod=mw_quote_addwatchlist)
Premarket
Last Updated: Oct 11, 2024 at 7:55 a.m. EDT Delayed quote
$
224.58
-14.19 -5.94%

**Comment by Xmabel112Lubi** [comment]
I think ower there its Teslas popular

Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Xmabel112Lubi** [comment]
Only I can't contribute because apparently my body can't handle it, even if my soul wants to
So I don't know yet what fate and what I still have to do to become everything he needs for me


Odoslané z aplikácie Outlook pre Android<https://aka.ms/AAb9ysg>
________________________________
